packages/cardano-graphql/src/operations/networkInfo.graphql

```diff
@@ -0,0 +1,23 @@
+query NetworkInfo {
+  queryEpoch(order: { desc: number }, first: 1) {
+    number
+    startedAt {
+      date
+    }
+    # Review: what's 'live stake'? We have it as part of NetworkInfo.stake (StakeSummary)
```

Comment:
_Live_ stake is the current stake participating in the protocol, _active_ stake is the snapshot being used to determine the current leadership schedule. For example, Alice delegates 100 ada to a pool in epoch 100. This stake is considered _live_. In 3 epochs this would be considered _live_.